File: pset1/mario/mario.c
#include<cs50.h>
#include<stdio.h>
int main(void)
{
    int height, i, j, k, l; // Declaration of variables
    // Take height as input from the user
    do
    {
        height = get_int("Height: ");
    }
    while ((height < 1) || (height > 8));

    l = height;

    //i indicates the number of rows
    for (i = 0; i < height; i++)
    {
        //k indicates the number of spaces in each row
        for (k = l - 1; k > 0; k--)
        {
            printf(" ");
        }

        //j indicates the number of left hashes in each row
        for (j = 0; j <= i; j++)
        {
            printf("#");
        }

        printf("  ");

        //this j indicates the number of right hashes in each row
        for (j = 0; j <= i; j++)
        {
            printf("#");
        }

        printf("\n");
        l--;
    }
}

File: pset6/mario.py
from cs50 import get_int

height = 0
# Get input from user
while height < 1 or height > 8:
    height = get_int("Height: ")

l = height
for i in range(height):
    k = l-1
    # print spaces
    while(k > 0):
        print(" ", end="")
        k -= 1
    # print hashes
    for j in range(i+1):
        print("#", end="")
    print()
    l -= 1